src/authorization/AllowEverythingAuthorizer.ts

```diff
@@ -1,10 +1,15 @@
+import type { Authorization } from './Authorization';
 import { Authorizer } from './Authorizer';
 
 /**
  * Authorizer which allows all access independent of the identifier and requested permissions.
  */
 export class AllowEverythingAuthorizer extends Authorizer {
-  public async handle(): Promise<void> {
-    // Allows all actions
+  public async handle(): Promise<Authorization> {
+    return {
+      addMetadata(): void {
```

Comment:
Actually, we probably want to generate `WAC-Allow` here as well, so return `WebAclAuthorization`.